build/includes/release.mk

```diff
@@ -87,3 +87,8 @@ pre-build-release:
 post-build-release:
 	docker run --rm $(common_mounts) -w $(workdir_path)/build/release $(build_tag) \
 		gcloud builds submit . --substitutions _VERSION=$(base_version) --config=./post_cloudbuild.yaml $(ARGS)
+
+# Ensure that ci/save_cache and ci/restore_cache are not deleted by the cleanup policy
+cache-save-restore:
```

Comment:
🤔 I don't need this for the setup of the Cloud Build scheduled triggers, but if we want to keep it for testing purposes (which is useful), I think it might be better to be part of https://github.com/googleforgames/agones/blob/main/build/includes/build-image.mk since it's not a target needed for the release process.

Open to suggestion on a better place, but that's the best idea I had.

Reply:
Understood. I'll go ahead and remove it since it's not required for the setup of the Cloud Build scheduled triggers.